### ISSUE TITLE
Helm 3: Missing values from debug output

### ISSUE THREAD

**Issue by endzyme** [helm/helm#6238]
As discussed in Developer Call on 2019-08-15: I think a refactor of how releases are printed may have removed the `COMPUTED VALUES:` and `USER-SUPPLIED VALUES:` sections of the output. 

Helm 2: https://github.com/helm/helm/blob/f4a55985b6d101409583067ca69018cd014cbf28/cmd/helm/printer.go#L33-L35 (Is used [by template](https://github.com/helm/helm/blob/master/cmd/helm/template.go#L185) and [by install](https://github.com/helm/helm/blob/master/cmd/helm/install.go#L449))

Helm 3: 
Refactors printing responsibility to [pkg/action](https://github.com/helm/helm/blob/f73e3235c2ef8931274898588d65b7802b56d265/pkg/action/printer.go#L30), which doesn't print values anymore.

Observations:
`pkg/action PrintRelease()` is only passed a `Release` struct, which doesn't have the the context of `USER-SUPPLIED VALUES:` anymore. This is due to the coalesce which  merges the Chart values during the install run (see [pkg/action/install.go](https://github.com/helm/helm/blob/f73e3235c2ef8931274898588d65b7802b56d265/pkg/action/install.go#L138)). By the time the `Run` method of `Install` struct has run, it loses the original values supplied.

Suggested change:
Adding a [`Release`](https://github.com/helm/helm/blob/f73e3235c2ef8931274898588d65b7802b56d265/pkg/release/release.go#L22) struct field for `UserSuppliedValues` or potentially `OriginalConfig` to match the existing `Config` naming. This new field would house the original map rendered from [`values.Options{}.MergeValues()`](https://github.com/helm/helm/blob/f73e3235c2ef8931274898588d65b7802b56d265/pkg/cli/values/options.go#L41).

Open to more suggestions and discussion though!

* * *
## Issue Template Info
Output of `helm version`: 
```
version.BuildInfo{Version:"v3.0.0-alpha.2", GitCommit:"97e7461e41455e58d89b4d7d192fed5352001d44", GitTreeState:"clean", GoVersion:"go1.12.7"}
```

Output of `kubectl version`: 
```
Client Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.1", GitCommit:"4485c6f18cee9a5d3c3b4e523bd27972b1b53892", GitTreeState:"clean", BuildDate:"2019-07-18T09:18:22Z", GoVersion:"go1.12.5", Compiler:"gc", Platform:"linux/amd64"}
```

Cloud Provider/Platform (AKS, GKE, Minikube etc.): 
N/A

**Comment by mattfarina** [comment]
This should not have been removed. PRs welcome to restore it.

**Comment by thomastaylor312** [comment]
I'm going to work on fixing this

**Comment by thomastaylor312** [comment]
So looking around at the code, I think the output change was very intentional, but it probably was not discussed very well. The `--debug` flag as it stands now actually outputs all the logging and isn't tied to outputting other kinds of data. It seems like the debug flag now has a conflated meaning as it has to do more with additional logging/context and not with the final output of a command. So, my question is whether we want another flag to indicate we want this output or if we should just include it by default? For now I am going to make it work with `--debug`, but wanted to raise this point of conversation

@endzyme Are the original values something you'd want to actually be stored in the release (like in the backing secret)? I would prefer not to because that would start to balloon out the size of the release object (which already get kind of big), so my thinking was to have it be on the object in memory, but not actually write it out to storage. Would that be an ok solution?

**Comment by thomastaylor312** [comment]
@endzyme So I am pretty sure the `Config` data member of a release contains the final merge of the user supplied values based on my initial experimentation. Am I missing something?

**Comment by endzyme** [comment]
Thanks for following up - yes - I believe I opened the issue before it was fixed in `dev-v3`. Thanks again!

**Comment by thomastaylor312** [comment]
Closed by #6550
